cassandra/src/main/scala/filodb.cassandra/columnstore/CassandraColumnStore.scala

```diff
@@ -169,11 +170,8 @@ extends CachedMergingColumnStore with CassandraColumnStoreScanner with StrictLog
     }
   }
 
-  def unwrapTokenRanges(wrappedRanges : Seq[TokenRange] ): HashSet[TokenRange] = {
-    val tokenRanges = new HashSet[TokenRange]()
-    wrappedRanges.map(range => tokenRanges.addAll(range.unwrap()))
-    logger.debug(s"unwrapTokenRanges: ${tokenRanges.toString()}")
-    tokenRanges
+  def unwrapTokenRanges(wrappedRanges : Seq[TokenRange] ): Seq[TokenRange] = {
```

Comment:
@parekuti two minor changes:  remove the space before ): and the braces { } is not needed since this is a one liner.  Not a big deal...